superset-frontend/src/components/GridTable/index.tsx

```diff
@@ -216,21 +231,16 @@ function GridTable<RecordType extends object>({
         `}
       >
         <AgGridReact
+          // TODO: migrate to Theme API - https://www.ag-grid.com/react-data-grid/theming-migration/
+          theme="legacy"
```

Comment:
with this theme value, the existing ag-theme-quartz styles are broken.
![_DEV__Superset](https://github.com/user-attachments/assets/48cf5d8c-746c-4715-b395-c5ce0ed74f3c)

Please migrate to theme api first to apply the theme name here.